src/Management/src/EndpointCore/Health/HealthRegistrationsAggregator.cs

```diff
@@ -19,25 +19,52 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading.Tasks;
 using HealthCheckResult = Steeltoe.Common.HealthChecks.HealthCheckResult;
 
 namespace Steeltoe.Management.Endpoint.Health
 {
-    public class HealthRegistrationsAggregator : DefaultHealthAggregator, IHealthRegistrationsAggregator
+    public class HealthRegistrationsAggregator : IHealthRegistrationsAggregator
     {
-        public HealthCheckResult Aggregate(IList<IHealthContributor> contributors, IOptionsMonitor<HealthCheckServiceOptions> healthServiceOptions, IServiceProvider serviceProvider)
+        private DefaultHealthAggregator _aggregator;
+        private DefaultAsyncHealthAggregator _asyncAggregator;
+
+        public HealthRegistrationsAggregator()
+        {
+            _aggregator = new DefaultHealthAggregator();
+            _asyncAggregator = new DefaultAsyncHealthAggregator();
+        }
+
+        public async Task<HealthCheckResult> Aggregate(IEnumerable<IHealthContributor> contributors, IEnumerable<IAsyncHealthContributor> asyncContributors, IOptionsMonitor<HealthCheckServiceOptions> healthServiceOptions, IServiceProvider serviceProvider)
```

Comment:
Is there a test to verify that the aggregation is as expected from an async contributor and non-async contributor?